vms/platformvm/service_test.go

```diff
@@ -365,6 +365,10 @@ func TestGetBalance(t *testing.T) {
 	require := require.New(t)
 	service, _, _ := defaultService(t)
 
+	staticFeesCfg := service.vm.Config.StaticConfig
```

Comment:
Since the fee config is embedded I don't think we need this var

Reply:
dropped